### ISSUE TITLE
vanguard 

### ISSUE THREAD

**Issue by ab220700** [KebsCS/KBotExt#240]
version 14.9.1, has this stopped working on the vanguard patch? when i try use the log in feature it says launching client but nothing happens

**Comment by icyrs** [comment]
it does still work for me, but rarely will get a vanguard error in lobby which will force quit the client. so it still works but keep in mind they may improve vanguard in the future to detect it fully

**Comment by ab220700** [comment]
> it does still work for me, but rarely will get a vanguard error in lobby which will force quit the client. so it still works but keep in mind they may improve vanguard in the future to detect it fully

Does the log in feature work with multiple accounts? When I try to click it says authentication failed

**Comment by ab220700** [comment]
> it does still work for me, but rarely will get a vanguard error in lobby which will force quit the client. so it still works but keep in mind they may improve vanguard in the future to detect it fully

when i click on my account to log in it says launching client but doesnt open the riot games exe

**Comment by KebsCS** [comment]
> when i click on my account to log in it says launching client but doesnt open the riot games exe


Are you on prerelease? https://github.com/KebsCS/KBotExt/releases/tag/prerelease
Issue similar to yours has been fixed it in before

**Comment by ab220700** [comment]
> 
> 
> > when i click on my account to log in it says launching client but doesnt open the riot games exe
> 
> 
> 
> 
> 
> Are you on prerelease? https://github.com/KebsCS/KBotExt/releases/tag/prerelease
> 
> Issue similar to yours has been fixed it in before
> 
> 

Yes I am on the pre-release. Since vanguard was added clicking on the username doesn't launch the riot client

**Comment by ab220700** [comment]
> > when i click on my account to log in it says launching client but doesnt open the riot games exe
> 
> Are you on prerelease? https://github.com/KebsCS/KBotExt/releases/tag/prerelease Issue similar to yours has been fixed it in before

This has been happening since the new vanguard update, so after the prelease update
